### DISCUSSION TITLE
Unable to Deploy 0.50.9 - Manifest Not Found

### DISCUSSION THREAD

**Discussion by BoostN** [airbytehq/airbyte#28552] [Connector Questions] [answered]
![image](https://github.com/airbytehq/airbyte/assets/6150217/73321324-2207-4050-b998-e130cd27d67a)

Error response from daemon: manifest for airbyte/airbyte-api-server:0.50.9 not found: manifest unknown: manifest unknown

Clean install from a fresh git clone. Any ideas?

**Comment by marcosmarxm** [comment, marked as answer]
This was fixed in `0.50.10` please pull the latest version of docker and env files.
